django_celery_results/backends/database.py

```diff
@@ -276,6 +275,10 @@ def on_chord_part_return(self, request, state, result, **kwargs):
                     callback=callback,
                     group_result=deps
                 )
+                chord_counter.delete()
```

Comment:
is it possible to add some test to verify this change?

Reply:
@auvipy i have to see if i can find time for that. Anyhow, we have a patched version running successfully for months now.

Reply:
but the tests in the CI are failing